src/transformers/models/vision_encoder_decoder/configuration_vision_encoder_decoder.py

```diff
@@ -52,15 +52,15 @@ class VisionEncoderDecoderConfig(PretrainedConfig):
     Examples:
 
     ```python
-    >>> from transformers import BertConfig, ViTConfig, VisionEncoderDecoderConfig, VisionEncoderDecoderModel
+    >>> from transformers import VisionEncoderDecoderModel, VisionEncoderDecoderConfig, ViTConfig, BertConfig
```

Comment:
Similar to #19580, no need to change this order here.

Reply:
updated.